### PR TITLE
Rename Acme example to Proprietary and modify package to add the pref…

### PR DESCRIPTION
**Pull request by sgilda**
…ix com.example.

### CODE REVIEW

File: proprietary-javaee-servlet/rules-xml/proprietary-servlet-annotations.windup.xml [hunk 1]
```diff
@@ -33,15 +33,15 @@
                 </hint>
             </perform>
         </rule>
-        <rule id="acme-servlet-annotations_002">
+        <rule id="proprietary-servlet-annotations_002">
```
Comment (lincolnthree) [inline proprietary-javaee-servlet/rules-xml/proprietary-servlet-annotations.windup.xml]: Should be: proprietary-servlet-annotations-02000

File: proprietary-javaee-servlet/rules-xml/proprietary-servlet-annotations.windup.xml [hunk 2]
```diff
@@ -1,30 +1,30 @@
...
             <addon id="org.jboss.windup.rules,windup-rules-java,2.0.0.Final"/>
         </dependencies>
-        <sourceTechnology id="acme" versionRange="(10,12]"/>
+        <sourceTechnology id="proprietary" versionRange="(10,12]"/>
         <targetTechnology id="eap" versionRange="[6,)"/>
         <tag>reviewed-2015-05-27</tag>
     </metadata>
     <rules>
-        <rule id="acme-servlet-annotations_001">
+        <rule id="proprietary-servlet-annotations_001">
```
Comment (lincolnthree) [inline proprietary-javaee-servlet/rules-xml/proprietary-servlet-annotations.windup.xml]: Should be: proprietary-servlet-annotations-01000

File: proprietary-javaee-servlet/rules-xml/proprietary-servlet-annotations.windup.xml [hunk 3]
```diff
@@ -50,15 +50,15 @@
                 </hint>
             </perform>
         </rule>
-        <rule id="acme-servlet-annotations_003">
+        <rule id="proprietary-servlet-annotations_003">
```
Comment (lincolnthree) [inline proprietary-javaee-servlet/rules-xml/proprietary-servlet-annotations.windup.xml]: Should be: proprietary-servlet-annotations-03000

File: proprietary-javaee-servlet/rules-xml/proprietary-servlet-annotations.windup.xml [hunk 4]
```diff
@@ -1,30 +1,30 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <ruleset xmlns="http://windup.jboss.org/v1/xml" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://windup.jboss.org/v1/xml http://windup.jboss.org/schema/windup-jboss-schema.xsd"
-    id="acme-servlet-annotations">
+    id="proprietary-servlet-annotations">
     <metadata>
         <!--  
         <description>
```
Comment (lincolnthree) [inline proprietary-javaee-servlet/rules-xml/proprietary-servlet-annotations.windup.xml]: This should be un-commented.

File: proprietary-javaee-servlet/rules-xml/proprietary-servlet-annotations.windup.xml (hunk 1 above)
Comment (sgilda) [inline proprietary-javaee-servlet/rules-xml/proprietary-servlet-annotations.windup.xml]: @lincolnthree : Are these related rules supposed to be 1000 apart? I thought it would be -01000, -01001, -01002 since they are related rules. Then the next unrelated batch would be -02000. If we space them all 1000 apart, what's the point of the '000' position?

**Comment by sgilda** [comment]
@lincolnthree : Are these related rules supposed to be 1000 apart? I thought it would be -01000, -01001, -01002 since they are related rules. Then the next unrelated batch would be -02000. If we space them all 1000 apart, what's the point of the '000' position?

**Comment by jsight** [comment]
@sgilda Yes, we don't want to have to renumber everything if rules are placed in between them.

**Comment by sgilda** [comment]
Updated rule ids in loud protest. ;-)

This should be ready now.
